docsource/usage/default_values.md

```diff
@@ -0,0 +1,65 @@
+```{toctree}
+:maxdepth: 2
+```
+
+# Default values
+
+Testing SQL queries can often involve repetitive setup for mock tables. In SQLMock, one effective way to streamline this process is by using default values. By setting reasonable defaults, you can significantly reduce the boilerplate code in your tests, especially when dealing with multiple input tables or complex queries. Let’s explore how you can efficiently implement this.
+
+## Utilizing Default Values in MockTable Fields
+
+Defining default values at the field level in your mock tables is straightforward. The default argument in the field definition allows you to set default values, which are particularly useful for ensuring that joins and other query functionalities operate correctly.
```

Comment:
**Suggestion:**
For the sentence:

> The **default argument in the field definition allows you to set default values**, which are particularly useful for ensuring that joins and other query functionalities operate correctly.

The bolded section above I had to read 3x to understand. I would consider extending to something like:

> The default argument in the field definition allows you to set default values consistency across all test scenarios in one step. They are particularly useful for ensuring that joins and other query functionalities operate correctly.